scripts/lint-packages.sh

```diff
@@ -477,6 +477,17 @@ lint_package() {
 			fi
 			unset urls_ok
 
+			# Check for GitHub archive URLs that should use /archive/refs/tags/
+			for url in "${TERMUX_PKG_SRCURL[@]}"; do
+				if [[ "$url" =~ github\.com/.*/archive/[^/]+\.(tar\.gz|zip)$ ]] &&
```

Comment:
Now there's an extra set of `]]`.
Which was an oversight with my previous suggested patch.
```suggestion
				if [[ "$url" =~ github\.com/.*/archive/[^/]+\.(tar\.gz|zip)$ &&
```

That causes errors along the lines of:
```console
./scripts/lint-packages.sh: line 483: https://github.com/mevdschee/2048.c/archive/v1.0.3.tar.gz: No such file or directory
```